src/optimizations/mergeInstancedMeshes.ts

```diff
@@ -1,61 +1,60 @@
 // Copyright 2019 Cognite AS
 
 import Sector from '../Sector';
-import { MergedMesh, MergedMeshMappings } from '../geometry/MergedMeshGroup';
+import { MergedMesh } from '../geometry/MergedMeshGroup';
 import * as THREE from 'three';
-import { InstancedMeshCollection } from '../geometry/InstancedMeshGroup';
 import SceneStats from '../SceneStats';
-import { TreeIndexNodeIdMap } from '../parsers/parseUtils';
 
 const globalMatrix = new THREE.Matrix4();
 
 const TRIANGLE_COUNT_LIMIT = 10000;
-
-function countMappingsToMerge(collections: InstancedMeshCollection[]) {
-  let numMappings = 0;
-  collections.forEach(collection => {
-    const triangleCount = collection.mappings.count * collection.triangleCount;
-    if (triangleCount < TRIANGLE_COUNT_LIMIT) {
-      numMappings += collection.mappings.count;
-    }
-  });
-  return numMappings;
-}
+type MappingInfo = {collectionIndex: number, mappingIndex: number, size: number};
```

Comment:
MappingInfo or MappingData? You typically don't sort info, but data? Or maybe I'm being silly?

Reply:
Personally, I only call things 'Data' if I'm going to be keeping it around for a while.

Reply:
Google to the rescue 😂. "Data are simply facts or figures — bits of information, but not information itself. When data are processed, interpreted, organized, structured or presented so as to make them meaningful or useful, they are called information. Information provides context for data.". ... I'd say it's info?

Reply:
Since I'm technically processing the data to make it useful

Reply:
Like, I'd never stick this in a database, so it's not data

Reply:
Sure!